src/components/UserProfile.jsx

```diff
@@ -18,6 +19,8 @@ import MetadataCard from './cards/MetadataCard';
 import SightingsCard from './cards/SightingsCard';
 import CollaborationsCard from './cards/CollaborationsCard';
 import CardContainer from './cards/CardContainer';
+import UserManagersCollaborationEditTable from '../pages/userManagement/UserManagerCollaborationEditTable';
```

Comment:
Because this component is no longer being used only within the user management page, it should be moved to a different directory.

Reply:
Done